assume/common/units_operator.py

```diff
@@ -248,7 +248,9 @@ def handle_data_request(self, content: DataRequestMessage, meta: MetaDict) -> No
 
         data = []
         try:
-            data = self.units[unit].outputs[metric_type][start:end]
+            data = (
+                self.units[unit].outputs[metric_type].as_pd_series(start=start, end=end)
+            )
```

Comment:
this is an internal process of ASSUME, so hopefully this is fine like this since changing it would need to call other functions from FAstSeries, which wouldn't make the situation clearer